README.md

```diff
@@ -139,8 +139,11 @@ In addition, user objects are embedded in the `_embedded['wp:authorship']` field
 
 ## WP-CLI
 
+Authorship implements a custom flag for use with posts, and a migration command for PublishPress Authors.
```

Comment:
Probably don't need to mention PPA directly here, since the migration command can expand in the future.